hack/netpol-generator/README.md

```diff
@@ -0,0 +1,43 @@
+# Cyclonus: Network policy generator
+
+The [Cyclonus](https://github.com/mattfenwick/cyclonus) tool generates test scenarios involving network policies,
+runs the scenarios on kubernetes clusters measuring connectivity between pods, and compares to measure results to
+expected results.  The result is a multidimensional conformance matrix of network policy features.
```

Comment:
I know that both are valid, but as a convention, in this repo, we typically use a single whitespace after a period

Reply:
👍